.travis.yml

```diff
@@ -13,13 +13,13 @@ install:
 before_script:
   - gem install awesome_bot
 script:
-  - allowed_internal_links=ternjs
+  - allowed_internal_links=ternjs,melpa
   - allowed_redirect_links=awesome.re
   - awesome_bot README.org --allow-ssl --white-list $allowed_internal_links,$allowed_redirect_links
   - cp README.org README.org.original
-  - $EMACS README.org -Q -nw --load gen-toc.el --eval '(progn (awesome-emacs-gen-toc)(save-buffer)(kill-emacs))'
+  - $EMACS README.org -Q -nw --eval "(progn (setq package-archives '((\"melpa\" . \"http://melpa.milkbox.net/packages/\"))) (package-initialize) (when (not (package-installed-p 'toc-org)) (when (not package-archive-contents) (package-refresh-contents)) (package-install 'toc-org)) (require 'toc-org) (toc-org-insert-toc) (save-buffer) (kill-emacs))"
```

Comment:
@snosov1 why is the melpa URL listed here at http://melpa.milkbox.net/packages instead of https://melpa.org/packages/

Reply:
Actually, no idea. I got this URL from my config that is a couple of years old. If `https://melpa.org/packages/` is the "correct" URL, I can change that.